content/python/concepts/inheritance/inheritance.md

```diff
@@ -68,28 +68,6 @@ Python here!
 
 **Note**: Inside the child class, the same number and type of arguments need to be passed into the parent class' method.
```

Comment:
```suggestion
> **Note:** Inside the child class, the same number and type of arguments need to be passed into the parent class' method.
```

Reply:
Also, near the top on line # 16, can we **bold** the word "Inheritance"?

Reply:
Updated.